frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -673,22 +673,21 @@ class Worksheet extends React.Component {
                 let elementHeight = element.height();
                 let screenScrollTop = $(window).scrollTop();
                 let screenHeight = $(window).height();
-                let scrollIsAboveElement = elementOffsetTop + elementHeight - screenScrollTop >= 0;
+                // 170 is the sum of height of WorksheetHeader and NavBar.
+                // Since they block the user's view, we should take them into account when calculating whether the item is on the screen or not.
+                let scrollIsAboveElement =
+                    elementOffsetTop + elementHeight - screenScrollTop - 170 >= 0;
```

Comment:
Does this change have to do with this PR ("remove the usage of string refs")?

Reply:
Not exactly. I just created a new PR #3023 to make it clearer!